kbs/Cargo.toml

```diff
@@ -7,7 +7,7 @@ documentation.workspace = true
 edition.workspace = true
 
 [features]
-default = ["coco-as-builtin", "coco-as-grpc", "intel-trust-authority-as"]
+default = ["coco-as-builtin", "coco-as-grpc", "intel-trust-authority-as", "pkcs11"]
```

Comment:
We can include this in the `--features` flag whenever it is needed, but we do not need to mark it as `default`